faunadb/field_test.go

```diff
@@ -69,3 +85,16 @@ func assertFailToExtractField(t *testing.T, value Value, field Field, message st
 	var res Value
 	require.EqualError(t, value.At(field).Get(&res), message)
 }
+
+func assertFailToExtractAnyFieldFor(t *testing.T, values ...Value) {
```

Comment:
Is this function supposed to be reused? Seems to be used only once in `TestFailToExtractFieldForNonTransversableValues`